CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Functions `getFilterState` to get current status of filters and `updateFilters` to update filters and refresh dataTable
```

Comment:
```suggestion
- Added function `getFilterState` to get current status of filters
- Added function `updateFilters` to update filters and refresh dataTable
```
what do you think? just to have every new feature on a separate line and it matches the previous style